packages/fx-core/src/common/tools.ts

```diff
@@ -119,13 +119,25 @@ export function objectToConfigMap(o?: Json): ConfigMap {
 }
 
 const SecretDataMatchers = [
+  "solution.localDebugTeamsAppId",
+  "solution.teamsAppTenantId",
   "fx-resource-aad-app-for-teams.clientSecret",
   "fx-resource-aad-app-for-teams.local_clientSecret",
+  "fx-resource-aad-app-for-teams.local_clientId",
+  "fx-resource-aad-app-for-teams.local_objectId",
+  "fx-resource-aad-app-for-teams.local_oauth2PermissionScopeId",
+  "fx-resource-aad-app-for-teams.local_tenantId",
+  "fx-resource-aad-app-for-teams.local_applicationIdUris",
   "fx-resource-simple-auth.filePath",
   "fx-resource-simple-auth.environmentVariableParams",
   "fx-resource-local-debug.*",
   "fx-resource-bot.botPassword",
   "fx-resource-bot.localBotPassword",
+  "fx-resource-bot.localBotId",
+  "fx-resource-bot.localObjectId",
+  "fx-resource-bot.local_redirectUri",
```

Comment:
Though related logic should be changed when multi-env support is ready. Is there any way to guarantee resource plugin developer adds local debug related settings here? Or it's just a temporary solution, and we no longer need to consider this when multi-env support is ready?

Reply:
Here is temporary solution to enable collaboration for users in local scenario. The logic will change after multi-env is supported.